.gitignore

```diff
@@ -4,6 +4,5 @@ node_modules
 .idea
 build-sizes.json
 data
-**/.env
 f2/build
-**/.env
+platform/**/.env.*
```

Comment:
We probably want to have .env.arango.example file, for example, in github though. Any way to include them but exclude the rest?

Reply:
Those .env.example files are used by [dotenv-safe](https://www.npmjs.com/package/dotenv-safe) to ensure that we have loud, descriptive failures when env vars are missing.
We will still have them in other folders but nothing uses those in the platform folder.